src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -208,5 +212,32 @@ public void WriteException(string errorMessage)
                 Console.WriteLine(log);
             }
         }
+
+        public static void Summarize(AnalysisSummary summary)
+        {
+            if (s_telemetryClient != null)
+            {
+                s_telemetryClient.TrackEvent(SummaryEventName, properties: new Dictionary<string, string>
+                {
+                    { "toolName", summary.ToolName },
+                    { "toolVersion", summary.ToolVersion },
+                    { "sessionId", s_sessionId },
+                    { "normalizedPath", summary.NormalizedPath },
+                    { "symbolPath", summary.SymbolPath },
+                    { "numberOfBinaryAnalyzed", summary.FileAnalyzed.ToString() },
+                    { "numberOfBinaryNotAnalyzed", summary.FileNotAnalyzed.ToString() },
```

Comment:
let's remove this for now.
If we say '0' it will be wrong.

Reply:
removed for now